yt_dlp/extractor/beeg.py

```diff
@@ -84,23 +76,12 @@ def _real_extract(self, url):
             })
         self._sort_formats(formats)
 
-        title = traverse_obj(video, ('file', 'stuff', 'sf_name'))
-        video_id = compat_str(video_id)
-        display_id = traverse_obj(video, ('fc_facts', 0, 'id'))
-        description = traverse_obj(video, ('file', 'stuff', 'sf_story'))
-        timestamp = unified_timestamp(traverse_obj(video, ('fc_facts', 0, 'fc_created')))
-        duration = int_or_none(traverse_obj(video, ('file', 'fl_duration')))
-
-        tags = [tag.get('tg_name') for tag in video['tags']] if video.get('tags') else None
-
         return {
             'id': video_id,
-            'display_id': display_id,
-            'title': title,
-            'description': description,
-            'timestamp': timestamp,
```

Comment:
Why did you remove timestamp and display_id!?

Reply:
Because It gets that from the first item of Json. ([fc_facts][0][id], or fc_created) it works when they are immutable. 

![image](https://user-images.githubusercontent.com/216170/152388217-88745a0e-44e9-476e-83c9-6b933bfcd809.png)

But for some videos this items changes. And we don't get a reliable information.

For example:
Yesterday the first item of the same video (https://beeg.com/-0599050563103750) had the id 2569975.
Today the first item is 2569969, and the 2569975 is on second item.
![image](https://user-images.githubusercontent.com/216170/152390120-8113cc76-fa1c-4f4a-8091-5d6c96768b9a.png)

I removed it from return because tests keeps failing for the same correct video.

If there is a way to ignore these keys from tests and keep it in return, it is ok to bring it back.

Reply:
No, dont return wrong values! You can check the id of each matching entry and return if it matches. But I'll let you decide whether to implement it

Reply:
I sorted items to get always the lower id, probably, the first (oldest) item. 
Now it should return always the same display_id and timestamp.